src/views/Login/index.tsx

```diff
@@ -27,12 +33,15 @@ const Login = () => {
   };
 
   const onSuccess = (tokens: any) => {
-    oktaAuth.handleLoginRedirect(tokens);
+    const referringUri = oktaAuth.getOriginalUri();
+    oktaAuth.handleLoginRedirect(tokens).then(() => {
+      history.push(referringUri);
+    });
   };
 
   useEffect(() => {
     if (authState.isAuthenticated) {
-      history.replace('/');
+      history.replace(redirect);
```

Comment:
This will allow us to send links to the GRT folks that look like `easi.cms.gov/signin?redirect=actual/url`